src/content/docs/getting-started/basics/index.md

```diff
@@ -0,0 +1,209 @@
+
+## Basic assets
+
+Global website assets are stored in the `src/assets` folder. Any files placed in this folder will be recursively copied to the root directory of the final website.
+
+```sh
+# Inside the src/content directory
+mkdir assets
+cd assets
+mkdir images
+cd images
+# Pick a logo and copy to the images directory
+cp ~/my-logo.png ./logo.png
+```
+
+The minimal theme uses the `logo.png` file as the main image for the website. This file must be placed inside the `images` folder. 
+
+It is also possible to create additional directories and place images, JavaScript, CSS, and other files in the `assets` folder. These files will be available as global assets when the website is built.
+
+Toucan differentiates between page assets and global assets. Each subpage can have its own assets directory, and together, the subpage and its assets are referred to as page bundles.
+
+## The 404 page
+
+Creating a custom "Not Found" page is a common use case for a website. To do this, create a special `404` directory inside the `contents` folder:
+
+```sh
+# Inside the src/content directory
+mkdir 404
+cd 404
+```
+
+The starting point for the 404 page is an `index.md` Markdown file. Create this file with the following contents:
+
+```markdown
+---
+title: "Not found"
+description: "Page not found"
+---
+
+# Not found
+
+![Not Found](./assets/not-found.png)
+
+This page does not exists.
+
+[Take me home](/)
+```
+
+This page includes a "not found" image, which needs to be placed inside the `404/assets` directory. As mentioned earlier, pages can have their own assets stored within the page bundle, which, in this case, is the `404` directory.
+
+```sh
+# Inside the src/content/404 directory
+mkdir assets
+cp ~/not-found.png ./not-found.png
+```
+
+Regenerate the website and refresh the browser window. Ensure that the browser cache is disabled to view the changes properly. Enter an invalid URL to verify that the system redirects to the custom 404 page.
+
+## Page bundles
+
+Like the 404 page, every subpage of the website can be defined as a page bundle. Page bundles are simple directories where the slug of the subpage matches the folder name containing the `index.md` file.
+
+Create a new `about` subpage by setting up a directory named `about` and placing an `index.md` file inside it.
+
+```sh
+# Inside the src/content directory
+mkdir about
+```
+
+Create an `index.md` file inside the `about` directory with the following contents:
+
+```markdown
+---
+title: "About"
+description: "About my website"
+---
+
+# About
+
+This page is all about my website.
+```
+
+Customize the contents of the about page as desired. Regenerate the site or use the `watch` command to monitor for changes and preview the updates in the browser.
+
+
+## Site navigation
+
+The final step of this tutorial is setting up a basic menu for the website. The minimal theme supports navigation, and new menu items can be added by defining them in the site's configuration file as shown below:
```

Comment:
Does this theme support navigation? I do not see the `navigation.mustache` file.

Reply:
yes, but it's inside the `blocks/header.mustache` file.

Reply:
fixed it in the theme to make this work: https://github.com/toucansites/minimal-theme/pull/1